java/src/jmri/jmrit/display/controlPanelEditor/shape/configurexml/PositionableShapeXml.java

```diff
@@ -110,30 +110,34 @@ public void loadCommonAttributes(PositionableShape ps, int defaultLevel, Element
 
         ps.setDisplayLevel(getInt(element, "level"));
 
-        Attribute a = element.getAttribute("hidden");
-        if ((a != null) && a.getValue().equals("yes")) {
-            ps.setHidden(true);
-            ps.setVisible(false);
+        try {
+            boolean value = element.getAttribute("hidden").getBooleanValue();
+            ps.setHidden(value);
+            ps.setVisible(!value);
+        } catch (DataConversionException e1) {
+            log.warn("unable to convert positionable shape hidden attribute");
+        } catch (Exception e) {
         }
-        a = element.getAttribute("positionable");
-        if ((a != null) && a.getValue().equals("true")) {
-            ps.setPositionable(true);
-        } else {
-            ps.setPositionable(false);
+
+        try {
+            ps.setPositionable(element.getAttribute("positionable").getBooleanValue());
+        } catch (DataConversionException e1) {
+            log.warn("unable to convert positionable shape positionable attribute");
+        } catch (Exception e) {
         }
 
-        a = element.getAttribute("showtooltip");
-        if ((a != null) && a.getValue().equals("true")) {
-            ps.setShowToolTip(true);
-        } else {
-            ps.setShowToolTip(false);
+        try {
+            ps.setShowToolTip(element.getAttribute("showtooltip").getBooleanValue());
+        } catch (DataConversionException e1) {
+            log.warn("unable to convert positionable shape showtooltip attribute");
+        } catch (Exception e) {
         }
 
-        a = element.getAttribute("editable");
-        if ((a != null) && a.getValue().equals("true")) {
-            ps.setEditable(true);
-        } else {
-            ps.setEditable(false);
+        try {
+            ps.setEditable(element.getAttribute("editable").getBooleanValue());
+        } catch (DataConversionException e1) {
+            log.warn("unable to convert positionable shape editable attribute");
+        } catch (Exception e) {
```

Comment:
We should not be catching all Exceptions and ignoring them. I know there is some code in JMRI that does this,  it that code should not be used as a model. Please remove all catches of Exception from these files in this PR. You can use multicatch syntax if desired (NetBeans will help with this).

Reply:
There's a few more "catch (Exception e)"'s in the LayoutEditorXml.save() method… but they're around some complicated layout editor save code (that I didn't write). I'm not going to touch those (unless someone more knowledgable than I can explain the right thing to do).